core.js

```diff
@@ -292,6 +299,13 @@ class FileTypeParser {
 			};
 		}
 
+		if (this.checkString('avro')) {
```

Comment:
Can you link to a source for this?

Reply:
I don't see it written anywhere there that the file header must start with `avro`.

Reply:
I've also noticed people use .avsc and .acme, but I don't quite understand the difference